### DISCUSSION TITLE
how do I get the current testcase render data in callback function?

### DISCUSSION THREAD

**Discussion by spwpun** [jtpereyda/boofuzz#632] [Q&A]
I want to write a callback function to process the current testcase data then use it to fuzz test.
**how do I get the current testcase render data in callback function?**
The origin callback fucntion seems to can't get it:
![image](https://user-images.githubusercontent.com/32606457/178639939-9a003ad0-47ef-4f6a-9766-8eacb023da70.png)
the test_case_context arg is a ProtocolSession, but it just contains the node, not contains the mutation_context:
![image](https://user-images.githubusercontent.com/32606457/178640273-a82e61ea-ee75-45c3-a35c-a4a866522b54.png)
Then if we need to get the current mutation testcase data, we must know the mutation_context:
![image](https://user-images.githubusercontent.com/32606457/178640457-fe9bbb37-dfff-4436-ab04-d1c1417bea39.png)

**Comment by newbing9527** [comment]
Have you solved this issue? I’m facing the same problem.
